project/profiles/capacity4more/modules/c4m/content/c4m_content/c4m_content.module

```diff
@@ -124,6 +131,55 @@ function c4m_content_form_node_form_alter(&$form, &$form_state, $form_id) {
 }
 
 /**
+ * Returns path to content page.
+ *
+ * @param string $content_type
+ *   The name of content type.
+ *
+ * @return string
+ *   Path to group content page
+ *   If $content_type is not group ('group'/'project') or group content,
+ *   defaults to site homepage.
+ */
+function c4m_content_construct_path_to_content_page($content_type) {
+  $group_content_types = array_keys(c4m_og_get_all_group_content_bundles('node'));
+
+  // Content type is not group content.
```

Comment:
this comment doesn't contribute to the understanding of the logic, please fix this.